app/routes/content/index.js

```diff
@@ -235,6 +235,11 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
     },
 
     setupController(controller, model) {
+        model.get('reviewLogs').then(reviewLogs => {
```

Comment:
`setupController` doesn't feel like the right place to put this. There is no guarantee that the promise will be resolved by the time the view loads. Maybe `afterModel`?